dwbb/src/interface/DW_pulse_sync.scala

```diff
@@ -0,0 +1,55 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2024 Jiuyang Liu <liu@jiuyang.me>
+package org.chipsalliance.dwbb.interface.DW_pulse_sync
+
+import chisel3._
+import chisel3.experimental.SerializableModuleParameter
+import upickle.default
+
+object Parameter {
+  implicit def rw: default.ReadWriter[Parameter] =
+    upickle.default.macroRW[Parameter]
+}
+
+case class Parameter(
+    regEvent: Boolean = true,
+    fSyncType: String = "pos_pos_sync",
+    tstMode: String = "no_latch",
+    verifEn: Int = 1,
+    pulseMode: Int = 0
```

Comment:
What's `verifEn` and `pulseMode`? They should be String type?

Reply:
It would be clearer to be an `Int` type. There is no stander string to represent these values in the original documents.

Reply:
`pulseMode` is one of 'single_src_domain_pulse', 'rising_dest_domain_pulse', 'falling_dest_domain_pulse', 'rising_falling_dest_domain_pulse'.